website/docs/3-Guidelines/1-typography.mdx

```diff
@@ -19,57 +19,57 @@ We set 16px (1rem) as the base font size for body text to ensure readability. It
 #### Heading 1
 |         | Font |Font size                                    | Line height                                   | Example |
 | ------- | ---  |-------------------------------------------- | --------------------------------------------- |---
-| mobile  | Roboto     |    30px or 1.875rem |  45px or 2.8rem    | <p style={{fontSize:1.875+'rem',lineHeight:2.8+'rem',fontFamily:'Roboto'}}>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-| mobile  | Archivo     |    30px or 1.875rem |  45px or 2.8rem    | <p style={{fontSize:1.875+'rem',lineHeight:2.8+'rem',fontFamily:'Archivo'}}>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-| desktop |Roboto|48px or 3rem        | 72px or 4.5rem |  <p style={{fontSize:3+'rem',lineHeight:4.5+'rem',fontFamily:'Roboto'}}>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-| desktop |Archivo|48px or 3rem        | 72px or 4.5rem |  <p style={{fontSize:3+'rem',lineHeight:4.5+'rem',fontFamily:'Archivo'}}>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+| mobile  | Roboto     | 3XL <br/> 30px <br/> 1.875rem | LH-3XL <br/> 45px <br/> 2.8rem    | <p style={{fontSize:1.875+'rem',lineHeight:2.8+'rem',fontFamily:'Roboto'}}>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
```

Comment:
@g-stamatis please use for all design tokens an increasing numeric system only
See for example open props
https://open-props.style/#typography
All of their properties are from 0 to x where 0 is a smaller number and x is the larger number.
In the case of font sizes, I would rather see font-size-6 or whatever number is needed rather than 3xl.

This way you learn the system once and you apply it for any design token without having to look
up what is the special naming that is needed for that token.